src/Uno.UWP/Storage/Streams/DataWriter.cs

```diff
@@ -0,0 +1,188 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Windows.Foundation;
+
+namespace Windows.Storage.Streams
+{
+	public partial class DataWriter : IDataWriter, IDisposable
+	{
+		private readonly List<byte> _unstoredBuffer = new List<byte>();
+
+		public DataWriter()
+		{
+
+		}
+
+		public UnicodeEncoding UnicodeEncoding { get; set; }
+
+		public ByteOrder ByteOrder { get; set; }
+
+		public uint UnstoredBufferLength => (uint)_unstoredBuffer.Count;
+
+		public void WriteByte(byte value) => _unstoredBuffer.Add(value);
+
+		public void WriteBytes(byte[] value) => _unstoredBuffer.AddRange(value);
+
+		public void WriteBuffer(IBuffer buffer)
+		{
+			if (buffer is InMemoryBuffer inMemory)
+			{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove or fill this block of code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)